build_tools/bazel/iree.bazelrc

```diff
@@ -15,6 +15,7 @@ build --experimental_guard_against_concurrent_changes
 build --compilation_mode=opt
 # Used in TensorFlow, so we have to enable it here as well.
 common --experimental_repo_remote_exec
+build --experimental_cc_shared_library
```

Comment:
Looks like we don't need this, because it's added in integrations/tensorflow/build_tools/bazel/iree-tf.bazelrc?

Reply:
yes we can remove it. Do you want me to update this PR?

Reply:
I'll do it. I was waiting CI bots. It looks fine so far!